core/modules/qana/TablePlugin.cc

```diff
@@ -267,14 +275,28 @@ TablePlugin::applyPhysical(QueryPlugin::Plan& p,
 }
 
 
+std::string TablePlugin::_getNextValueExprAlias() {
+    return "valueExprAlias_" + std::to_string(_nextValueExprAlias++);
```

Comment:
I'm not very familiar with plugins, is this code thread safe (or is it called from other protected code)?

Reply:
it's single threaded at this point.